gradle.properties

```diff
@@ -2,13 +2,13 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties
-minecraft_version=1.20.2
-yarn_version=1.20.2+build.4
-loader_version=0.14.24
+minecraft_version=1.20.4
+yarn_version=1.20.4+build.3
+loader_version=0.15.2
```

Comment:
Not sure what the "correct" loader version to use is, I went with the same version meteor-client is using https://github.com/MeteorDevelopment/meteor-client/blob/master/gradle.properties#L6